### PR TITLE
Support animation of SVG color properties

### PR DESCRIPTION
**Pull request by ewilligers**
Animation of 'fill', 'flood-color', 'lighting-color', 'stop-color',
'stroke' is now supported.
https://github.com/web-animations/web-animations-js/issues/81

### CODE REVIEW

File: test/js/color-handler.js [hunk 1]
```diff
@@ -15,10 +15,10 @@ suite('color-handler', function() {
   });
   test('color interpolation', function() {
     assert.equal(webAnimations1.propertyInterpolation('color', '#00aa11', '#aa00bb')(0.2), 'rgba(34,136,51,1)');
-    assert.equal(webAnimations1.propertyInterpolation('color', 'transparent', '#004488')(0), 'transparent');
-    assert.equal(webAnimations1.propertyInterpolation('color', 'transparent', '#004488')(0.5), 'rgba(0,68,136,0.500)');
-    assert.equal(webAnimations1.propertyInterpolation('color', 'red', 'green')(2), 'rgba(0,255,0,1)');
-    assert.equal(webAnimations1.propertyInterpolation('color', 'red', 'green')(-1), 'rgba(255,0,0,1)');
+    assert.equal(webAnimations1.propertyInterpolation('fill', 'transparent', '#004488')(0), 'transparent');
```
Comment (suzyh) [inline test/js/color-handler.js]: I think we need to use something other than 0 for the progress on this test, to distinguish successful interpolation from a 50-50 flip.

File: test/js/color-handler.js [hunk 2]
```diff
@@ -15,10 +15,10 @@ suite('color-handler', function() {
...
+    assert.equal(webAnimations1.propertyInterpolation('stroke', 'transparent', '#004488')(0.5), 'rgba(0,68,136,0.500)');
+    assert.equal(webAnimations1.propertyInterpolation('lightingColor', 'red', 'green')(2), 'rgba(0,255,0,1)');
+    assert.equal(webAnimations1.propertyInterpolation('textDecorationColor', 'red', 'green')(-1), 'rgba(255,0,0,1)');
```
Comment (suzyh) [inline test/js/color-handler.js]: Do we need to test all the different properties? If so, we seem to be missing some.

File: test/js/color-handler.js (hunk 1 above)
Comment (ewilligers) [inline test/js/color-handler.js]: The subsequent line has the same from/to colors with 50% progress.

File: test/js/color-handler.js (hunk 2 above)
Comment (ewilligers) [inline test/js/color-handler.js]: I have updated the test to indicate SVG color properties are being tested.

**Comment by suzyh** [comment]
lgtm

**Pull request merged**